tests/data/csv-sample.csv

```diff
@@ -0,0 +1,2 @@
+0,1,2,A
+1,2,3,B
```

Comment:
Does this file serve any purpose?  I don't see it used in any of the tests.

Reply:
@dav3r - nice catch.. this is irrelevant file for this pull-request.
File removed - dc78243